[IMP] install: troubleshoot typical problems for Mac users (#2495)
odoo/documentation: content/administration/install/install.rst

Odoo uses PostgreSQL as database management system. Use `postgres.app <https://postgresapp.com>`_
to download and install PostgreSQL (supported version: 10.0 and later).

> .. warning:: For the command line tools bundled with `postgres.app` to be available, make sure to
>           setup your `$PATH` variable by following
>           `these instructions <https://postgresapp.com/documentation/cli-tools.html>`_.

Reviewer: - it isn't a warning. Maybe a note or a tip ?
- We start the content after the admonition command (see [RST cheat sheet: tip alert block](https://www.odoo.com/documentation/15.0/contributing/documentation/rst_cheat_sheet.html#tip))
- You can go to the line anywhere there is a space, even inside the URL. (see [RST guidelines](https://www.odoo.com/documentation/15.0/contributing/documentation/rst_guidelines.html#start-a-new-line-before-the-100th-character))
- it's better to write a relevant tag for your hyperlink, rather than "this link", "here", or "these instructions

```suggestion
.. tip::
   To make the command line tools bundled with `postgres.app` available, make sure to setup your
   `$PATH` variable by following the `Postgress.app CLI Tools instructions
   <https://postgresapp.com/documentation/cli-tools.html>`_.
```

Odoo uses PostgreSQL as database management system. Use `postgres.app <https://postgresapp.com>`_
to download and install PostgreSQL (supported version: 10.0 and later).

.. tip::

>    To make de command line tools bundled with `postgres.app` available, make sure to setup your

Reviewer 2: ```suggestion
   To make the command line tools bundled with `postgres.app` available, make sure to setup your
```
?

Where `CommunityPath` is the path of the Odoo Community installation
and `mydb` is the default database to serve on `localhost:8069`.

> .. warning:: Some versions of python on Mac might lead to the following error message:
>           `current limit exceeds maximum limit`. In that case, add the following
>           parameter to avoid the problem: `--limit-memory-hard 0`.

Reviewer: ```suggestion
.. warning::
   Some versions of python on Mac might lead to the following error message: `current limit exceeds
   maximum limit`. In that case, add the following parameter `--limit-memory-hard 0` to avoid the
   problem.
```
